packages/parser/src/schemas/sqs.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from 'zod';
+
+const SqsMsgAttributeSchema = z.object({
+  stringValue: z.string().optional(),
+  binaryValue: z.string().optional(),
+  stringListValues: z.array(z.string()).optional(),
+  binaryListValues: z.array(z.string()).optional(),
+  dataType: z.string(),
+});
+
+const SqsAttributeSchema = z.object({
+  ApproximateReceiveCount: z.string(),
+  ApproximateFirstReceiveTimestamp: z.string(),
+  MessageDeduplicationId: z.string().optional(),
+  MessageGroupId: z.string().optional(),
+  SenderId: z.string(),
+  SentTimestamp: z.string(),
```

Comment:
The timestamp is a unix timestamp, while zod can only parse string with iso format. We'd need to run a `transform` for all timestamp fields and I'd like to avoid it for now. If there is customer demand we will add it.

Reply:
Agree for all the unix ones, let's leave them as numbers for now.

The ones that are ISO format I'd still switch to datetime though